viewing_party/party.py

```diff
@@ -1,23 +1,160 @@
-# ------------- WAVE 1 --------------------
+from collections import Counter
+import copy
 
 def create_movie(title, genre, rating):
-    pass
+    # If title, genre, or rating are "None", return None, otherwise create movie
+    if title == None or genre == None or rating == None:

```

Comment:
Great; this check works, though it is more customary to use `is` to check whether a variable is None. [This article](https://www.pythontutorial.net/advanced-python/python-none/) gives a good explanation.